pyproject.toml

```diff
@@ -24,3 +24,9 @@ file = "src/imars3d/_version.py"
 pythonpath = [
   ".", "src"
 ]
+testpaths = ["tests"]
+python_files = ["*test*.py"]
```

Comment:
should this be `["test*.py"]` to enforce a pattern where only modules with prefix `test` run?